package-lock.json

```diff
@@ -6,20 +6,20 @@
   "dependencies": {
```

Comment:
Why do you have dependencies changes if you did not install new packages? Recall to use `npm ci` rather than `npm install` unless you intentionally want to change dependencies